### PR TITLE
S6/front end

### PR DESCRIPTION
**Pull request by dpellenwood**
Builds out the plugin FE display. Uses plain-Jane CSS & JS. No compilers, PCSS, ES6, or other build tools.

https://central.tri.be/issues/70597

### CODE REVIEW

File: assets/css/cuny-branding-bar.css [hunk 1]
```diff
@@ -1,12 +1,239 @@
...
+	padding: 0;
+}
+
+.cuny-branding-bar__logo a {
+	position: relative;
+	display: block;
+	height: 42px;
+	width: 220px;
+	margin: 0 10px;
+	background: transparent url('/wp-content/plugins/cuny-branding-bar/assets/img/logo.png') center no-repeat;
```
Comment (jbrinley) [inline assets/css/cuny-branding-bar.css]: This is not a reliable URL. Can it be relative to the stylesheet path?

File: assets/js/cuny-branding-bar.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+ */
+
+jQuery(function($) {
+	init($);
+});
+
+/**
+ * Initialize the CUNY Branding bar scripts.
+ */
+function init($) {
```
Comment (jbrinley) [inline assets/js/cuny-branding-bar.js]: You've put the `init` function in the global namespace.

File: src/Branding_Bar_Service_Provider.php [hunk 3]
```diff
@@ -33,11 +33,18 @@ private function register_templates( Container $container ) {
 		};
 		$container[ 'template.nav_menu_args' ] = function( Container $container ) {
 			return [
-				'theme_location' => $container[ 'nav.menu.key' ],
+				'theme_location'    => $container[ 'nav.menu.key' ],
+			    'fallback_cb'       => false,
+			    'container'         => false,
+			    'depth'             => 1,
+			    'items_wrap'        => '<ol>%3$s</ol>',
```
Comment (jbrinley) [inline src/Branding_Bar_Service_Provider.php]: Whitespace.

File: assets/css/cuny-branding-bar.css (hunk 1 above)
Comment (dpellenwood) [inline assets/css/cuny-branding-bar.css]: Not sure how I'd do that as this isn't compiled from any preprocessor and we're not using the theme. It's all internal to the plugin. I'm open to suggestions though.

Comment (jbrinley) [inline assets/css/cuny-branding-bar.css]: `background: transparent url( '../img/logo.png' ) center no-repeat;`

Comment (dpellenwood) [inline assets/css/cuny-branding-bar.css]: Duh. Not sure where my brain was on that one. Thank you.

**Comment by jbrinley** [comment]
👍

File: assets/js/cuny-branding-bar.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+		 * Set the state values on init and resize
+		 */
+		function setState() {
+			state.v_height = document.body.clientHeight;
+			state.v_width = document.body.clientWidth;
+			state.is_mobile = (state.v_width < state.mobile_breakpoint);
+		}
+
+		/**
+		 * Simple debounce function from underscores.js
```
Comment (graemehoffman) [inline assets/js/cuny-branding-bar.js]: I like this decision to only use what you need. Smart.

File: views/branding-bar.php [hunk 5]
```diff
@@ -3,7 +3,30 @@
...
+	itemtype="https://schema.org/SiteNavigationElement"
+	itemscope
+>
+	<h1 class="cuny-branding-bar__logo">
+		<a href="https://www.journalism.cuny.edu/">
+			<span class="visually-hide"><?php _e( 'CUNY Graduate School of Journalism', 'tribe' ); ?></span>
+		</a>
+	</h1>
+
+	<button id="cunybar__toggle" class="cuny-branding-bar__toggle" aria-expanded="false" aria-label="<?php _e( 'Toggle Sites Menu' , 'tribe' ); ?>">
```
Comment (graemehoffman) [inline views/branding-bar.php]: Is it worth putting in some aria properties to explicitly connect this toggle button to the menu? Ex: aria-controls aria-haspopup

File: assets/css/cuny-branding-bar.css [hunk 6]
```diff
@@ -1,12 +1,239 @@
...
+	padding: 0;
+}
+
+.cuny-branding-bar__logo a {
+	position: relative;
+	display: block;
+	height: 42px;
+	width: 220px;
+	margin: 0 10px;
+	background: transparent url('../img/logo.png') center no-repeat;
```
Comment (dpellenwood) [inline assets/css/cuny-branding-bar.css]: Unfortunately, the comps I was given don't contain the logo in vector format. It's a hacked combination of different rendered layers (in the PSD). Regardless, I made sure that it is always served at 2x so we don't have to worry about it being blurry.

**Pull request merged**